src/openzaak/sass/admin/components/_autorisatie-form.scss

```diff
@@ -58,4 +58,16 @@ $color-row-background: #f9f9f9 !default;
   & &__va-selection {
   }
 
+  div.external-type ul {
```

Comment:
All the styling so far is done with BEM and the sass operator `&` - please refactor to use proper CSS classnames.

Don't target HTML elements, only target class names please.